File: Assignment2_3.java
//Accept on number from user if number is less than 10 then print " HEllo"otherwise print "Demo"
import java.util.*;

class Assignment2_3
{

  public static void main(String Args[])
  {
    Scanner sobj = new Scanner(System.in);
    System.out.println("Enter the Number:");
    int iNo = sobj.nextInt();

    Demo dobj = new Demo();

    dobj.Display(iNo);
    sobj.close();
  }
}
class Demo
{
  public int iValue;
  Demo()
  {
    iValue = 0;
  }

  void Display(int iValue)
  {
    if(iValue < 10)
    {
      System.out.println("Hello");
    }
    else
    {
      System.out.println("Demo");
    }

  }
}

File: Assignment2_3.cpp
//Accept on number from user if number is less than 10 then print " HEllo"otherwise print "Demo"

#include<iostream>
using namespace std;

class Demo
{
  public:
    int iNo;

    Demo()
    {
      iNo = 0;
    }
    void Display(int iNo)
    {
      if(iNo < 10)
      {
        cout<<"Hello\n";
      }
      else
      {
        cout<<"Demo\n";
      }
    }
};

int main()
{
  int iValue = 0;

  cout<<"Enter the Number:\n";
  cin>>iValue;

  Demo dobj;

  dobj.Display(iValue);


  return 0;
}